Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM alpine:3.2
 
-RUN apk add -U \
-	ca-certificates \
-	py-pip \
-	python \
- && rm -rf /var/cache/apk/* \
- && pip install --no-cache-dir --upgrade \
-	pip \
-	setuptools \
-	wheel
+ARG GITSHA="VCS ref not found"
+ARG BUILDDATE="Build date not found"
+LABEL org.label-schema.url="https://github.com/drone-plugins/drone-pypi" \
```

Comment:
Please take the Dockerfile from the codecov plugin as an example. Multiple of the label values are already set by drone

Reply:
Done

Reply:
What about plugins/base image maybe switch to this one?

Reply:
Yeah, makes sense. This base image will be updated pretty soon.

Reply:
Done